README.md

```diff
@@ -15,18 +15,45 @@ Azure Cosmos DB is Microsoft’s globally distributed multi-model database servi
 ## Running this sample
 
 * Before you can run this sample, you must have the following prerequisites:
+    * [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) or [Azure Cloud Shell](https://shell.azure.com) - Install if you want to execute commands via the CLI instead of the Azure portal.

```

Comment:
Remove all en-us from all the links (this helps with localization)

Reply:
Let's also add somewhere in the top that it corresponds to this doc: https://docs.microsoft.com/azure/cosmos-db/create-sql-api-python.